gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/translator/DotNetTranslator.java

```diff
@@ -510,6 +510,7 @@ static final class SymbolHelper {
         private final static Map<String, String> FROM_CS_MAP = new HashMap<>();
 
         static {
+            TO_CS_MAP.put(GraphTraversal.Symbols.asString, "AsString<object>");
```

Comment:
If we don't need these added steps to be generic in C#, then we also don't need to add them to this map here since the translator can just translate `asString` to `AsString`.